index.js

```diff
@@ -8,6 +8,7 @@ var extname = require('path').extname;
 var basename = require('path').basename;
 var resolve = require('path').resolve;
 var testit = require('testit');
+var camelCase = require('camelcase')
```

Comment:
It looks like we're using all lowercase for our variable names. `camelCase` should likely follow that pattern with `camelcase`.